srfi-130-test.scm

```diff
@@ -885,9 +887,9 @@
     (fail 'string->list/cursors))
 
 (or (equal? '(#\b #\c)
-            (string->list/cursors "abc"
-                                  (string-index->cursor "abc" 1)
-                                  (string-index->cursor "abc" 3)))
+            (string->list/cursors ABC
```

Comment:
Hold it, this change is wrong. ABC could be unicode, which will not work because the expectation is ascii.